packages/react/src/useEditor.ts

```diff
@@ -99,7 +101,9 @@ export const useEditor = (options: Partial<EditorOptions> = {}, deps: Dependency
   useEffect(() => {
     let isMounted = true
 
-    editorRef.current = new Editor(options)
+    if (!editorRef.current) {
```

Comment:
Take a look at https://github.com/ueberdosis/tiptap/pull/4579 if you get the chance. That pull request uses memo instead of a ref and also eliminates the double initialization that also causes issues. Best not to create the editor instance in an effect, but a memo instead; that way it gets created first render and after any deps array changes.

Also heads up, this is _also_ similar to this draft PR by https://github.com/ueberdosis/tiptap/pull/4858/files, which has the same issue of double editor initialization

Reply:
pls 🙏 @bdbch  @svenadlung